lib/commands/gesture.js

```diff
@@ -11,7 +11,9 @@ commands.moveTo = iosCommands.gesture.moveTo;
 
 commands.click = async function (el) {
   if (!this.isWebContext()) {
-    throw new errors.UnknownError('Command should be proxied to WDA');
+    // there are multiple commands that map here, so manually proxy
+    el = util.unwrapElement(el);
+    return await this.proxyCommand(`/element/${el}/click`, 'POST', {});
```

Comment:
It is not much different, except takes the element from the post data rather than the query. But it's part of the Selenium spec so it needs to be there.